azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -202,10 +216,8 @@ private void retryFlowQuietly(ExecutableFlow flow, Status originalStatus) {
   private void deleteContainerQuietly(final int executionId) {
     try {
       this.containerizedImpl.deleteContainer(executionId);
-    } catch (final ExecutorManagerException eme) {
-      logger.error("ExecutorManagerException while deleting container.", eme);
-    } catch (final RuntimeException re) {
-      logger.error("Unexpected RuntimeException while deleting container.", re);
+    } catch (final Exception e) {
```

Comment:
catch Throwable.

Reply:
It seems Exception should be sufficient. The cleanup action is schedule in a frequent manner. A failure does not affect the performance.